data_models/summarizer_output.py

```diff
@@ -140,3 +166,10 @@ def get_stamp_type_score(self):
             return 10.0
         if self == self.MEDIA_WITH_TEXT_AND_TITLE:
             return 20.0
+
+    def get_stamp_types_with_overlay(self):
```

Comment:
```
def is_text_overlay_present(self):
return self == TEXT_ONLY or self == MEDIA_WITH_TEXT or self == MEDIA_WITH_TEXT_AND_TITLE
```
Similar to is_embedded_content in ContentType

Reply:
Updated accordingly! Thanks for suggestion.